core/src/main/scala/com/yahoo/maha/core/query/oracle/OracleQueryGenerator.scala

```diff
@@ -1030,21 +1030,21 @@ b. Dim Driven
       val hasPartitioningScheme = fact.annotations.contains(OracleQueryGenerator.ANY_PARTITIONING_SCHEME)
 
       //add subquery
-      if(isFactOnlyQuery) {
-        queryContext.dims.foreach {
-          subqueryBundle =>
-            val factFKCol = fact.publicDimToForeignKeyMap(subqueryBundle.publicDim.name)
-            val factFkColAlias = {
-              if (fact.columnsByNameMap.contains(factFKCol)) {
-                fact.columnsByNameMap.get(factFKCol).get.alias
-              } else {
-                None
-              }
-            }
-            val sql = generateSubqueryFilter(factFkColAlias.getOrElse(factFKCol), filters, subqueryBundle)
-            whereFilters += sql
-        }
-      }
+//      if(isFactOnlyQuery) {
```

Comment:
Commenting this out because subqueries for dim-fact query type are handled by ogb generator.

Reply:
Oracle OGB and OracleQueryGenerator are two different query generators, not sure why are you commenting existing functionality of non ogb query generator.

Reply:
This code is un-reachable for non-ogb query generator because of the change in ogb criteria, specifically how isSubQueryCandidate is computed. Earlier, this code would be reached for a fact-only query with filter on dim non-primaryKey but now this case gets redirected to ogb query generator. In cases where the query is fact-only without any dim non-primaryKey filters then queryContext.dims is empty and the for-loop is not entered. In any other cases, the query is not fact-only.